### PR TITLE
Port ovirt setup to ipi-install-install

### PR DESCRIPTION
**Pull request by Gal-Zaidman**
This PR ports ovirt to use the generic ipi-install-install and stableinstall, to do so we:
- Created a step for etcd ramfs, this step creates a MC that make the masters use tmpfs for etcd
- Create a step to send events to ovirt
- Edited the ipi-install-install to move stuff to shared dir when the job exits, this will make sure that files are always moved
- Modified the workflow and chain definitions.

### CODE REVIEW

File: ci-operator/step-registry/ipi/etcd/on-ramfs/ipi-etcd-on-ramfs-ref.metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"path": "ipi/etcd/on-ramfs/ipi-etcd-on-ramfs-ref.yaml",
```
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/etcd/on-ramfs/ipi-etcd-on-ramfs-ref.metadata.json:2]: nitpick: why not `ipi/conf/etcd-on-ramfs`?

Comment (Gal-Zaidman) [reply ci-operator/step-registry/ipi/etcd/on-ramfs/ipi-etcd-on-ramfs-ref.metadata.json:2]: you suggested ipi-etcd-on-ramfs as the step so I thought that is what you meant

Comment (vrutkovs) [reply ci-operator/step-registry/ipi/etcd/on-ramfs/ipi-etcd-on-ramfs-ref.metadata.json:2]: sorry, my fault. I've realized other platforms could use that, so its worth moving to shared config dir

**Review by vrutkovs — APPROVED** [review]
/lgtm

Needs a better PR description that `ipi-install-install-copy` is a temporary step to test `prepare-next-steps` change

**Comment by vrutkovs** [comment]
/lgtm
/hold
/cc @wking 

Looks fine, lets have Trevor review it as well

**Comment by Gal-Zaidman** [comment]
@wking when reviewing notice that you probably should only care about:
https://github.com/openshift/release/pull/15447/commits/f00712e055877e18192b008ada2310ad5f9ef9da
This change is required because you are setting "errexit" and if some command fails (link the install for example) then the pod will exit and the manifest.json will not be copied to shared dir, which will result in deprovision exiting:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh#L16-L19

File: ci-operator/step-registry/ipi/install/ovirt/send-event-to-ovirt-post/ipi-install-ovirt-send-event-to-ovirt-post-commands.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+source ${SHARED_DIR}/ovirt-lease.conf
+# shellcheck source=/dev/null
+source "${CLUSTER_PROFILE_DIR}/ovirt.conf"
+# shellcheck source=/dev/null
+source "${CLUSTER_PROFILE_DIR}/send-event-to-ovirt.sh"
+
+if [[ -e "${SHARED_DIR}/failure" ]]; then
+  send_event_to_ovirt "Failure"
+elif [[ -e "${SHARED_DIR}/success" ]]; then
+    send_event_to_ovirt "Success"
```
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/install/ovirt/send-event-to-ovirt-post/ipi-install-ovirt-send-event-to-ovirt-post-commands.sh:17]:
```suggestion
  send_event_to_ovirt "Success"
```
nitpick

**Comment by vrutkovs** [comment]
/hold cancel

Doesn't look like we have serious objections against this change, the rest can be fixed later

**Pull request merged**